charts/minio-manager/README.md

```diff
@@ -30,13 +32,14 @@ The following table lists the parameters of this chart.
 |`minioAdminCredentials.secretKeys.useSsl`        | Name of the secret key for use ssl field                                   |MINIO_USE_SSL               |
 |`image.repository`                               | Repository of the minio image                                              |minio/mc                    |
 |`image.pullPolicy`                               | Kubernetes image pull policy                                               |IfNotPresent                |
-|`image.tag`                                      | Tag of the minio image                                                     |RELEASE.2021-01-05T05-03-58Z|
+|`image.tag`                                      | Tag of the minio image                                                     |RELEASE.2020-11-25T23-04-07Z|
```

Comment:
I presume that newer MinIO client versions can work with older server versions (we know the opposite isn't true).
But either way it's great that this is now one of the many parameters in the chart 👍

Reply:
This is not the case for the policy creation. The error message explicitly state to use an oldest version of the client. It's between an api version v2 and v3 changes.

Reply:
Actually, I didn't try the version in the README against the server version deployed from the chart. I tested against our server version deployed in single configuration (the version from May). But it's correcting what is written in the value file against what is in this column.